openshift/job-build-dependencies-template.yaml

```diff
@@ -0,0 +1,156 @@
+    displayName: Log Path
+    required: false
+    value: "/opt/app-root/src/build.log"
+
+  - name: THOTH_BUILD_ANALYSER_OUTPUT_FORMAT
+    description: Required Output format.
+    displayName: ImageStream Tag
+    required: false
+    value: "json"
+
+  - name: IMAGE_STREAM_NAMESPACE
+    description: Project/Namespace to be used for deployment.
+    displayName: ImageStream Project
+    required: true
+
+  - name: THOTH_BUILD_ANALYSER_JOB_ID
+    description: A unique dentifier of build-analyser job.
+    displayName: Build Analyser id
+    required: true
+
+  - name: THOTH_BUILD_LOG_DOC_ID
+    description: Document id of Build log present in ceph.
+    displayName: Build Log Document ID
+    required: true
+
+  - name: THOTH_REPORT_OUTPUT
+    description: Remote where results should be send to.
+    displayName: ImageStream Tag
+    required: false
+
+objects:
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: ${THOTH_BUILD_ANALYSER_JOB_ID}
+      labels:
+        app: thoth
+        component: build-dependencies
+        operator: graph-sync
+        task: build-analyser
+        mark: cleanup
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          labels:
+            app: thoth
+            component: build-dependencies
+            mark: cleanup
+        spec:
+          restartPolicy: Never
+          automountServiceAccountToken: false
+          containers:
+            - name: build-dependencies
+              image: "${IMAGE_STREAM_REGISTRY}/${IMAGE_STREAM_NAMESPACE}/build-analyser:${IMAGE_STREAM_TAG}"
+              env:
+                - name: THOTH_BUILD_LOG_DOC_ID
+                  value: "${THOTH_BUILD_LOG_DOC_ID}"
+                - name: THOTH_BUILD_ANALYSER_LOG_PATH
+                  value: "${THOTH_BUILD_ANALYSER_LOG_PATH}"
+                - name: THOTH_BUILD_ANALYSER_OUTPUT_FORMAT
+                  value: "${THOTH_BUILD_ANALYSER_OUTPUT_FORMAT}"
+                - name: THOTH_REPORT_OUTPUT
+                  value: "${THOTH_REPORT_OUTPUT}"
+                - name: THOTH_BUILD_ANALYSER_SUBCOMMAND
+                  value: "dependencies"
+                - name: THOTH_DEPLOYMENT_NAME
+                  valueFrom:
+                    configMapKeyRef:
+                      key: storage-bucket-name
+                      name: thoth
+                - name: THOTH_S3_ENDPOINT_URL
+                  valueFrom:
+                    configMapKeyRef:
+                      key: ceph-host
+                      name: thoth
+                - name: THOTH_CEPH_BUCKET
+                  valueFrom:
+                    configMapKeyRef:
+                      key: ceph-bucket-name
+                      name: thoth
+                - name: THOTH_CEPH_BUCKET_PREFIX
+                  valueFrom:
+                    configMapKeyRef:
+                      key: ceph-bucket-prefix
+                      name: thoth
+                - name: THOTH_USER_API_APP_SECRET_KEY
```

Comment:
I think this env is specific to user-api. It is not needed in these yaml templates.

Reply:
copy-paste mistake, how can I get those :eyes:?